### PR TITLE
chore(spdx-utils): Upgrade the SPDX license list to version 3.23

### PR DESCRIPTION
**Pull request by fviernau**
The enums were regenerated using

./gradlew :utils:spdx-utils:generateSpdxEnums

### CODE REVIEW

File: utils/spdx/src/main/resources/licenses/BSD-3-Clause-HP [hunk 1]
```diff
@@ -6,7 +6,7 @@ are met:
 2. Redistributions in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution.
-3. Neither the name of the HP nor the names of its
+3. Neither the name of the  nor the names of its
```
Comment (sschuberth) [inline utils/spdx/src/main/resources/licenses/BSD-3-Clause-HP:9]: Ouch, this line actually looks like a mistake. Should we manually revert it?

File: utils/spdx/src/main/resources/licenses/BSD-3-Clause-Sun [hunk 2]
```diff
@@ -1,29 +1,25 @@
-Copyright (c) 2001-2013 Oracle and/or its affiliates. All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
 
-Redistribution and  use in  source and binary  forms, with  or without
-modification, are permitted provided that the following conditions are
-met:
+-Redistribution of source code must retain the above copyright notice, this
```
Comment (sschuberth) [inline utils/spdx/src/main/resources/licenses/BSD-3-Clause-Sun:4]: Hmm, the missing " " after "-" also looks like a mistake... same below.

File: utils/spdx/src/main/resources/licenses/MIT-testregex [hunk 3]
```diff
@@ -1,4 +1,4 @@
- * Permission is hereby granted, free of charge, to any person obtaining a
+* Permission is hereby granted, free of charge, to any person obtaining a
```
Comment (sschuberth) [inline utils/spdx/src/main/resources/licenses/MIT-testregex:1]: Also here indentation is off.

File: utils/spdx/src/main/resources/licenses/URT-RLE [hunk 4]
```diff
@@ -1,4 +1,4 @@
- * This software is copyrighted as noted below.  It may be freely copied,
+* This software is copyrighted as noted below.  It may be freely copied,
```
Comment (sschuberth) [inline utils/spdx/src/main/resources/licenses/URT-RLE:1]: Again indentation is off.

**Review by sschuberth — COMMENTED** [review]
How do we want to proceed with obvious degradations when importing a new version?

**Comment by fviernau** [comment]
> How do we want to proceed with obvious degradations when importing a new version?

I'd propose to import these anyway, and file an upstream issue or PR to fix it. So, it gets fixed in the next re-import.

File: utils/spdx/src/main/resources/licenses/BSD-3-Clause-HP (hunk 1 above)
Comment (sschuberth) [reply utils/spdx/src/main/resources/licenses/BSD-3-Clause-HP:9]: Reported: https://github.com/nexB/scancode-toolkit/issues/3664

File: utils/spdx/src/main/resources/licenses/BSD-3-Clause-Sun (hunk 2 above)
Comment (sschuberth) [reply utils/spdx/src/main/resources/licenses/BSD-3-Clause-Sun:4]: Reported: https://github.com/nexB/scancode-toolkit/issues/3665

File: utils/spdx/src/main/resources/licenses/MIT-testregex (hunk 3 above)
Comment (sschuberth) [reply utils/spdx/src/main/resources/licenses/MIT-testregex:1]: Reported: https://github.com/nexB/scancode-toolkit/issues/3666

File: utils/spdx/src/main/resources/licenses/URT-RLE (hunk 4 above)
Comment (sschuberth) [reply utils/spdx/src/main/resources/licenses/URT-RLE:1]: Added to the report at https://github.com/nexB/scancode-toolkit/issues/3666.

**Comment by sschuberth** [comment]
> file an upstream issue

I've done that now.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**